examples/cms-umbraco-heartcore/components/avatar.js

```diff
@@ -0,0 +1,8 @@
+export default function Avatar({ name, picture }) {
+  return (
+    <div className="flex items-center">
+      <img src={picture} className="w-12 h-12 rounded-full mr-4" alt={name} />
```

Comment:
Could we use `next/image`? Some of the other CMS examples will show this!

Reply:
Updated